client/landing/gutenboarding/hooks/use-steps.ts

```diff
@@ -15,7 +15,16 @@ import { usePlanFromPath } from './use-selected-plan';
 
 export default function useSteps(): Array< StepType > {
 	const locale = useLocale();
-	const { hasSiteTitle } = useSelect( ( select ) => select( ONBOARD_STORE ) );
+	const { hasSiteTitle, hasSelectedDesignWithoutFonts } = useSelect(
+		( select ) => {
+			const onboardSelect = select( ONBOARD_STORE );
+			return {
+				hasSiteTitle: onboardSelect.hasSiteTitle(),
+				hasSelectedDesignWithoutFonts: onboardSelect.hasSelectedDesignWithoutFonts(),
+			};
+		},
+		[ ONBOARD_STORE ]
+	);
```

Comment:
Optional: I feel this abstraction is kinda frivolous. I'd do this and call the functions in usage. 
```suggestion
	  const { hasSiteTitle, hasSelectedDesignWithoutFonts } = useSelect(
		  ( select ) => select( ONBOARD_STORE ),
		  [ ONBOARD_STORE ]
	  );
```

Reply:
I 100% agree that this code seems very frivolous, but I don't think that less verbose alternatives (including your suggestion) would potentially introduce bugs (see this amazing p2 for more explanation p7H4VZ-2Q6-p2 )

Reply:
Oh yeah I remember! Makes sense to call the function inside the hook.